### PR TITLE
Update buildbot

### PR DESCRIPTION
**Pull request by 0152la**
Mirror changes on buildbot side.
* update path to `qemu`
* do not assume argument is passed to script

### CODE REVIEW

**Comment by 0152la** [comment]
bors try

**Comment by 0152la** [comment]
bors try

**Comment by 0152la** [comment]
bors try-

**Comment by 0152la** [comment]
bors try

**Comment by 0152la** [comment]
bors try

**Comment by 0152la** [comment]
bors try

**Comment by 0152la** [comment]
bors try

File: .buildbot.sh [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
 # Note that `sdk` and `morello-sdk` contain different versions of clang-format,
 # so we have to pick one and use it consistently.
-CLANG_FORMAT="$CHERIBASE"/output/sdk/bin/clang-format
+CLANG_FORMAT=$CHERI_DIR/morello-sdk/bin/clang-format
 find . -iname "*.c" -o -iname "*.h" -o -iname "*.cpp" -o -iname "*.hpp" | xargs "$CLANG_FORMAT" --dry-run -Werror
 
 echo "Checking that all the purecap examples build for all platforms..."
-PLATFORMS='riscv64-purecap morello-purecap'
+#PLATFORMS='riscv64-purecap morello-purecap'
+PLATFORMS='morello-purecap'
```
Comment (ltratt) [inline .buildbot.sh:20]: Shouldn't we be testing more than just morello-purecap?

**Comment by 0152la** [comment]
This updates this repo's buildbot config to the "new" setup (and also tests the communication between bors and buildbot at the same time). The failing check is a spurious check, which I've now fixed on buildbot's end (finally, with help from Edd).

File: .buildbot.sh (hunk 1 above)
Comment (0152la) [reply .buildbot.sh:20]: Right, forgot to add this in. Our nightly doesn't yet build risc-v, so it'll take a while to rebuild and test.

Comment (ltratt) [reply .buildbot.sh:20]: No problem!

Comment (0152la) [reply .buildbot.sh:20]: While I'm here, do we also want to build RISC-V hybrid, or just purecap?

Comment (ltratt) [reply .buildbot.sh:20]: I think we previously only tested riscv-purecap? If so, that's good enough for now (we can add risc-v hybrid a bit later).

**Comment by 0152la** [comment]
bors try

File: .buildbot.sh (hunk 1 above)
Comment (0152la) [reply .buildbot.sh]: Re-added in https://github.com/capablevms/cheri-examples/pull/68/commits/8a34e3adaab6b1001017cf843f9a3d80e204caa7.

**Comment by ltratt** [comment]
Please squash.

**Comment by 0152la** [comment]
Squashed.

**Pull request merged**